Doc/add ai policy pointer (#14037)
mne-tools/mne-python: doc/development/contributing.rst

==================

.. highlight:: console

> .. important::

Reviewer: ```suggestion

.. important::
```

.. highlight:: console

.. important::

>    **AI Usage Policy:** Before submitting any code or documentation, please make sure to review our AI usage policy outlined in the main CONTRIBUTING.md file at the root of the repository.

Reviewer: ```suggestion
   **AI Usage Policy:** Before submitting any code or documentation, please make sure to review our AI usage policy outlined in the repository `CONTRIBUTING.md file <https://github.com/mne-tools/mne-python/blob/main/CONTRIBUTING.md`_.
```

>    **AI Usage Policy:** Before submitting any code or documentation, please make sure to review our AI usage policy outlined in the repository `CONTRIBUTING.md file <https://github.com/mne-tools/mne-python/blob/main/CONTRIBUTING.md`_.

Reviewer: ```suggestion
   **AI Usage Policy:** Before submitting any code or documentation, please make sure to review our AI usage policy outlined in the repository `CONTRIBUTING.md file <https://github.com/mne-tools/mne-python/blob/main/CONTRIBUTING.md>`_.
```
